Mesh_3/include/CGAL/Labeled_mesh_domain_3.h

```diff
@@ -428,7 +428,9 @@ class Labeled_mesh_domain_3
 
 #ifndef CGAL_NO_DEPRECATED_CODE
   template<typename Function, typename Bounding_object, typename CGAL_NP_TEMPLATE_PARAMETERS>
+#if !defined(BOOST_MSVC) || (BOOST_MSVC >= 1910)
   CGAL_DEPRECATED
+#endif
   Labeled_mesh_domain_3(const Function& function,
```

Comment:
That is a bug in Microsoft Visual C++ 2015:
https://stackoverflow.com/questions/38378693/did-visual-studio-2015-update-3-break-constructor-attributes

Here are two possible workarounds:
https://stackoverflow.com/questions/46018977/what-is-the-correct-syntax-for-deprecating-a-single-constructor-in-visual-studio

🤮

Reply:
Thanks! let's keep it that way then.